src/documentation/sdks/java-extensions.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: "Beam Java SDK Extensions"
+permalink: /documentation/sdks/java-extensions/
+---
+# Apache Beam Java SDK Extensions
+
+## <a name="join-library"></a>Join-library
+
+Join-library provides inner join, outer left and right join functions. The aim
```

Comment:
Is "outer left and right join" one thing, or is it two? a comma would help wherever is appropriate.
inner join, outer left join, and outer right join functions  ?